amaranth_types/types.py

```diff
@@ -58,7 +58,7 @@
 
 # Deprecated
 LayoutLike: TypeAlias = (
-    Layout | Sequence[tuple[str, "ShapeLike | LayoutLike"] | tuple[str, "ShapeLike | LayoutLike", Direction]]
+    Layout | Sequence[tuple[str, "ShapeLike | LayoutLike"] | tuple[str, "ShapeLike | LayoutLike"]]
```

Comment:
```suggestion
    Layout | Sequence[tuple[str, "ShapeLike | LayoutLike"]]
```
tuple is duplicated

Reply:
Technically as external project we should still support type-checking of deprecated types (by filtering `DeprecationWarning` on import like amaranth does internally), but `LayoutLike` is only used in `amaranth.hdl.rec` which is removed completely on current amarnanth `main`, so it doesn't matter to much will be removed on next update here.

Reply:
good catch